Applications/ProductionAssist/Client/ProductionAssistFeedbackClient.cs

```diff
@@ -22,17 +22,18 @@ public async Task<IEnumerable<FeedbackWorkstation>> GetWorkstations()
         }
 
         /// <inheritdoc />
-        public async Task ReportAsFinished(Guid workstationId, string productionEntityId, int quantity)
+        public async Task ReportAsFinished(Guid workstationId, string productionEntityId, int quantity, string publicId, string barcode)
```

Comment:
@HOMAG-Andreas-Ilg  What is a pubicId and why do I need a pubicId and a barcode?